testdata/sqllogictests/datatypes.slt

```diff
@@ -0,0 +1,10 @@
+# fixed size list round trip
+
+statement ok
+create table test as select arrow_cast([1,2], 'FixedSizeList(2, Int64)') as f;
```

Comment:
can you test:

- inserting into the table
- doing a `select *` query and getting the results
- doing some operation on the data (e.g. summing a few rows).